apollo-configservice/src/test/java/com/ctrip/framework/apollo/configservice/integration/ConfigControllerIntegrationTest.java

```diff
@@ -67,7 +67,7 @@ public void setUp() throws Exception {
     someDC = "someDC";
     someDefaultCluster = ConfigConsts.CLUSTER_NAME_DEFAULT;
     someClientIp = "1.1.1.1";
-    executorService = Executors.newSingleThreadExecutor();
```

Comment:
what's the difference between `newSingleThreadExecutor` and `newFixedThreadPool`?

Reply:
newSingleThreadExecutor may be shutdown by gc. see the https://bugs.openjdk.java.net/browse/JDK-8145304